tools/bq-queryplan-sankey/bq-queryplan-sankey.js

```diff
@@ -0,0 +1,160 @@
+// Copyright 2019 Google LLC
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+
+//     https://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+// This file prepares the BigQuery query plan JSON data 
+// into a format acceptable by D3JS' Sankey library
+
+// Check if stage is for repartitioning
+function isRepartitionStage(stageName) {
```

Comment:
should have JsDoc - http://usejsdoc.org/

Reply:
Fixed the file with [http://usejsdoc.org/](http://usejsdoc.org/) style comments.